core/src/test/java/org/apache/iceberg/rest/TestRESTUtil.java

```diff
@@ -139,4 +148,100 @@ public void testOAuth2FormDataDecoding() {
 
     assertThat(RESTUtil.decodeFormData(formString)).isEqualTo(expected);
   }
+
+  @ParameterizedTest
+  @MethodSource
```

Comment:
we typically use explicit method names here rather than naming the argument method and the test method the same. 
```suggestion
  @MethodSource("validRequestUris")
```